apps/archive/archive.py

```diff
@@ -443,8 +444,10 @@ def deschedule_item(self, updates, original):
         :param original: original is document.
         """
         updates[ITEM_STATE] = CONTENT_STATE.PROGRESS
-        updates[PUBLISH_SCHEDULE] = None
-        updates[SCHEDULE_SETTINGS] = {}
+        # we need to convert schedule time to UTC
+        tz = pytz.timezone(original[SCHEDULE_SETTINGS]['time_zone'] or 'UTC')
+        localized = tz.localize(original['publish_schedule'].replace(tzinfo=None))
+        updates[PUBLISH_SCHEDULE] = localized.astimezone(pytz.utc)
```

Comment:
@akintolga that's what I've done initially, but after doing some tests, I realized that hour was changing if I just keep original settings. I think the frontend modifying time according to time_zone (because the shift is equal to the time difference). So if I don't modify the time here, scheduled time will be changed on each deschedule.

An other option would be to not change time in frontend in case of descheduling. I did it here because I was feeling it like more safe, but I can do the other way if better.